parquet-tools/README.md

```diff
@@ -99,9 +99,10 @@ where option is one of:
     -c,--column <arg>  Dump only the given column, can be specified more than
                        once
     -d,--disable-data  Do not dump column data
-       --debug         Disable color output even if supported
+       --debug         Enable debug output
```

Comment:
thanks for fixing. I see the same error above. (line 85, 92)
either add -n in those paragraphs as well or rework a little to avoid repetition. 
(common options paragraph or something)